oauth2client/client.py

```diff
@@ -844,6 +834,19 @@ def _generate_refresh_request_headers(self):
 
         return headers
 
+    def _initialize_headers(self, headers):
```

Comment:
Please place this definition earlier in the file ("above") any of the places where it is used.